packages/commercetools/api-client/src/api/getShippingMethods/index.ts

```diff
@@ -1,16 +1,16 @@
 import { ApolloQueryResult } from 'apollo-client';
-import { apolloClient } from '../../index';
+import { apolloClient, locale } from '../../index';
 import defaultQuery from './defaultQuery';
 import { ShippingMethodQueryResult } from './../../types/GraphQL';
 
 interface ShippingMethodData {
   shippingMethods: ShippingMethodQueryResult;
 }
 
-const getShippingMethods = async (): Promise<ApolloQueryResult<ShippingMethodData>> => {
+const getShippingMethods = async (cartId?: string): Promise<ApolloQueryResult<ShippingMethodData>> => {
   return await apolloClient.query({
     query: defaultQuery,
-    variables: {},
+    variables: { locale, cartId },
```

Comment:
Just a question from me: Can a variable be undefined? Will it be removed from query in this case?

Reply:
not really, there is no undefined in graphql  queries, variable always  must be defined, but it can be empty - I marked this as an optional field for the future - other query will be used if id is not defined